README.md

```diff
@@ -100,137 +58,73 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **Has an animals table schema**
+- **Has animals table data**
+- **A couple of animals search queries**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
- [optional] The live demo section can be removed since the projects is not deployed. This will keep your README file clean.